tests/web/test_scope.py

```diff
@@ -69,7 +73,19 @@ async def test_scopes():
         "http://sub.perdu.com/page.html",
         "https://perdu.com/secure.html",
         "http://perdu.com/subdir/page.html?k=v",
-        "http://perdu.com/subdir/page.html"
+        "http://perdu.com/subdir/page.html",
+        "http://subdomain.perdu.com/",
+        "http://subdomain.perdu.com/page.html",
+        "http://subdomain.perdu.com/subdir/subdirpage.html",
+        "https://subdomain.perdu.com/"
+    }
+
+    scope = Scope(Request("http://subdomain.perdu.com/"), "subdomain")
```

Comment:
Logic seems good to me. Maybe use a full URL to make the test more explicit like what was done for the `domain` scope

`scope = Scope(Request("http://subdomain.perdu.com/subdir/page.html?k=v"), "subdomain")`